lib/board.rb

```diff
@@ -0,0 +1,28 @@
+class Board
+  attr_reader :header,
+              :rows,
+              :columns,
+              :grid
+
+  def initialize
+    @header = ["A", "B", "C", "D", "E", "F", "G"]
+    @rows = 6
+    @columns = 7
+    @grid = make_grid
+  end
+
+  def make_grid
+    Array.new(@rows, Array.new(@columns, "."))
+  end
+
+  def print_board
+    puts format_board
+  end
+
+  def format_board
```

Comment:
We'll need to revisit this method in the future. It could use some cleaning up. We also might be resetting the grid every time we call it. For the purpose of iteration 1, it is functional.